### DISCUSSION TITLE
Inheritance of custom fields for prefixes

### DISCUSSION THREAD

**Discussion by christianmasopust** [netbox-community/netbox#14619] [Q&A] [answered]
The title says it all, are custom fields set on a prefix inherited to its child prefixes?

**Comment by candlerb** [comment, marked as answer]
No. Nothing is inherited by a prefix from its parent. Not tenant, not role, not site, and not custom fields.

The only thing I know of in Netbox with an "inheritance-like" model is config contexts, but these can only be inherited from a pre-set list of object types which are ancestors of Device or Virtual Machine, and Prefix is not one of those.

**Reply by christianmasopust** [reply]
Thanks a lot for your quick reply!
